src/lib/selectors.ts

```diff
@@ -0,0 +1,623 @@
+import * as topics from './pubSub/topics';
+import type {
+  User,
+  UserUpdateParams,
+} from '@sendbird/chat';
+import { FailedMessageHandler, MessageHandler, UserMessage, UserMessageCreateParams } from '@sendbird/chat/message';
+import { GroupChannel, GroupChannelCreateParams, SendbirdGroupChat } from '@sendbird/chat/groupChannel';
+import { OpenChannel, OpenChannelCreateParams, SendbirdOpenChat } from '@sendbird/chat/openChannel';
+import { FileMessage, FileMessageCreateParams, FileMessageUpdateParams, SendableMessage, UserMessageUpdateParams } from '@sendbird/chat/lib/__definition';
```

Comment:
> FileMessage, FileMessageCreateParams, FileMessageUpdateParams, SendableMessage, UserMessageUpdateParams
arent these in `@sendbird/chat` ?

Reply:
Yes, you can check here https://sendbird.com/docs/chat/v4/javascript/ref/modules/_sendbird_chat_message.html